ops/ruby/devops-framework/README.md

```diff
@@ -0,0 +1,108 @@
+# DevOps Framework
+
+## Purpose
```

Comment:
Please update this to reflect what we discussed today. My takeaway understanding was:

This is an experiment in an approach to a devops framework. The rationale for it coexisting with the other devops tooling is that the current framework has many issues, and it would be a large undertaking to refactor the entirety of the existing system in one go. At a later point we will evaluate whether we want to switch our existing devops tooling to follow this model, or merge these models, or move to a separate third (e.g. non-Ruby) model.